tests/net/socket/tls/prj.conf

```diff
@@ -0,0 +1,40 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+#CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=16384
+
+CONFIG_NET_LOG=y
+CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=y
+CONFIG_MBEDTLS_DEBUG=y
+CONFIG_MBEDTLS_DEBUG_LEVEL=4
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_BUILTIN=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=60000
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=2048
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_TEST=y
+CONFIG_NET_LOOPBACK=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_TCP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_TLS_CREDENTIALS=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_SOCKETS_TLS_MAX_CONTEXTS=2
+
```

Comment:
The default net_pkt/net_buf count isn't enough for this test, I wasn't actually even able to connect at TCP level with the defaults. I've used the following values, but these could probably be finetuned:
```
CONFIG_NET_PKT_TX_COUNT=24
CONFIG_NET_BUF_TX_COUNT=64
```

Reply:
I wasn't sure exactly what was causing that!
Thank you!! It was driving me kind of crazy